src/Microsoft.Health.Dicom.Core/Features/Query/QueryParser.cs

```diff
@@ -164,6 +195,8 @@ private bool ParseFilterCondition(KeyValuePair<string, StringValues> queryParame
                 condition = valueParser(dicomTag, trimmedValue);
```

Comment:
should 'var tagTypeCode = dicomTag.DictionaryEntry.ValueRepresentations.FirstOrDefault()?.Code;' be updated for cusomtag? Since it should be retrieved from database instead.